docs/user/dashboard/lens-advanced.asciidoc

```diff
@@ -43,9 +43,9 @@ The list of fields that appear depend on the <<index-patterns,index pattern>>, <
 
 [discrete]
 [[view-the-number-of-transactions-per-day]]
-=== View the number of transactions per day
+=== View the number of transactions per hour
 
-To determine the number of orders made each day, create a bar chart, then add it to the dashboard.
+To determine the number of orders made per hour, create a bar chart, then add it to the dashboard.
```

Comment:
Can you add some explanation about the default interval vs scaled-down interval?

Reply:
Ahh, another problematic one, but definitely valuable. What do you think about adding something like "This will result in displaying count as rate per hour instead of calculating count in the span of time buckets of underlying interval (in this case 12 hours)"  in line 65?

Reply:
What about "This will calculate an hourly average for each 12-hour interval, which is less granular but faster than querying each hour."